js/index.js

```diff
@@ -1,51 +1,38 @@
-// Get the html container element
-const htmlContainerElement = document.getElementById('mainContent');
-
-// Modal elements
+// Getting DOM elements
 const modal = document.getElementById('myModal');
 const modalContent = document.getElementById('modalContent');
 const input = document.getElementById('searchInput');
+const htmlContainerElement = document.getElementById('mainContent');
 
-// Request/Response
-
-let httpRequest;
-let responseContent;
-
-// Received data storage
+// Global scope variables
 
 let parsed;
+let offset = 0;
+const limit = 35; //number of gifs to be displayed at once
```

Comment:
Why did you declare number of gifs in a separate variable?

Reply:
Because I wanted to make it easier to change the number of gifs to be displayed at once without having to change a lot of other variables. Thus, when I change the number of gifs (limit) the step also changes so the offset will be (limit + limit ... + limit) depending on iteration.

Reply:
However you are using limit directly rather than offset.
Generally thats a great approach and you should definitely use it so you don't have to change constants in different places all over your code/project. Just in that specific case offset becomes an extra variable and should be a constant, not a let